website/src/get-started/quickstart-java.md

```diff
@@ -111,6 +111,55 @@ d-----        7/19/2018  11:00 PM                subprocess
 -a----        7/19/2018  11:00 PM           7662 WordCount.java
 ```
 
+Here is how a basic implementation of WordCount looks like.
```

Comment:
same comments here that I left in the  Python walkthrough -- move to bottom in its own section?